hazelcast/src/test/java/com/hazelcast/cluster/AbstractJoinTest.java

```diff
@@ -35,7 +35,7 @@
 public class AbstractJoinTest extends HazelcastTestSupport {
 
     protected void testJoin(Config config) throws Exception {
-        config.setProperty(ClusterProperty.WAIT_SECONDS_BEFORE_JOIN.getName(), "0");
+        config.setProperty(ClusterProperty.WAIT_SECONDS_BEFORE_JOIN.getName(), "1");
```

Comment:
I ran several times `TcpIpJoinTest`, even with `WAIT_SECONDS_BEFORE_JOIN` set to `0` and with 1500millis delay in `ClusterServiceImpl#finalizeJoin` as described in https://github.com/hazelcast/hazelcast/issues/18751#issuecomment-863016090 and didn't get a failure. It's not a guarantee that it won't fail, but it seems the original circumstances under which this test used to fail no longer hold. wdyt about reverting this change to get a few more PR builder runs?

Reply:
Doesn't matter to me. I will do if it'll get this PR merged :)
Keeping behavior the same dictates 1 millisecond, but if the problem indeed went away due to some other changes,
0 will do just as well

Reply:
@jbartok what do you think?